src/reducers/index.js

```diff
@@ -12,9 +12,9 @@ const initialState = {
 export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
-    // Fill in the body of this case
+      return { count: state.count + 1 }
```

Comment:
Could also be written as `count: ++state.count`, no difference, just giving a different perspective.